src/sitemap.njk

```diff
@@ -4,11 +4,13 @@ eleventyExcludeFromCollections: true
 ---
 <?xml version="1.0" encoding="utf-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-    {% for page in collections.all %}
+    {% for page in collections.all -%}
```

Comment:
This change and the one on line 13 removes a lot of whitespace, sneaking it in in this PR. Sorry, not sorry :)